PhysicaloidLibrary/src/com/physicaloid/lib/usb/UsbAccessor.java

```diff
@@ -59,7 +59,7 @@ public void init(Context context) {
         }
 
         if(mPermissionIntent == null) {
-            mPermissionIntent = PendingIntent.getBroadcast(context, 0, new Intent("USB_PERMISSION"), 0);
+            mPermissionIntent = PendingIntent.getBroadcast(context, 0, new Intent("USB_PERMISSION"), PendingIntent.FLAG_IMMUTABLE);
```

Comment:
Add another if() checking current API level, and set a variable as-required for the flag, that will make it more API agnostic.

Reply:
Thank you for your suggestion! I will create new pull request with checking API level.